app/helpers/home_helper.rb

```diff
@@ -1,15 +1,17 @@
 module HomeHelper
   def data_count(user, type)
+    years = FormHandler.instance.lettings_in_crossover_period? ? [current_collection_start_year, previous_collection_start_year] : [current_collection_start_year]
```

Comment:
Do we ever have different periods for lettings and sales forms? It seems odd to be specifying lettings here but using it for both.

Reply:
I don't think we do, but we could so I've updated it